lib/drives/pages/drive_detail_page.dart

```diff
@@ -195,9 +198,25 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
         title: const Text('Drive Detail'),
         actions: <Widget>[
           IconButton(
-            onPressed: () {},
-            icon: const Icon(Icons.chat),
-          )
+            onPressed: () => Navigator.push(
+              context,
+              MaterialPageRoute(
+                builder: (context) => DriveChatPage(
+                  drive: _drive!,
+                ),
+              ),
+            ).then((value) => loadDrive()),
+            icon: Badge(
+              badgeContent: Text(
+                _drive?.pendingRides?.length.toString() ?? '',
+                style: const TextStyle(color: Colors.white),
+                textScaleFactor: 1.0,
+              ),
+              showBadge: _drive?.pendingRides?.isNotEmpty ?? false,
+              position: BadgePosition.topEnd(top: -12),
```

Comment:
Yes I was also really happy with how this turned out. The only problem for me is that you can see it go to 0, and then vanish completely when you finish your notifications in the chat view. Feels like showBadge is a little bit too slow or something.

Regarding the usage on Cards: I'm not sure if we can do that, a little dot like this is probably not noticeable enough. It would have to be something more obvious like a diagonal banner over the edge or a different background color, right?

Reply:
I actually haven't noticed the slowness, but it sounds to me like a cool effect :D
Haven't we talked about a shortcut to the chats on the cards when there are notifications even? In that case, I think the exact design can be copied. Otherwise, yeah a little dot is what I envisioned for the tab as well

Reply:
Let's declare it a feature